### PR TITLE
[FLINK-26744] Init document for table store and introduce quickstart

### PR DESCRIPTION
**Pull request by JingsongLi**
- Just like other sub project of Apache Flink, table store should build docs like Flink ML and Flink Stateful Functions.
- Introduce index page and quickstart page.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+2 - 4. The ToC can be omitted by adding the following to the front matter of
+the page:
+
+    ---
+    bookToc: false
+    ---
+
+### ShortCodes 
+
+Flink uses [shortcodes](https://gohugo.io/content-management/shortcodes/) to add
```
Comment (LadyForest) [inline docs/README.md:88]: Haha, do you forget to replace this? `Flink Table Store uses...`

File: docs/content/_index.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+# Apache Flink Table Store
+
+Flink Table Store is a unified streaming and batch store for building dynamic
+tables on Apache Flink. It is designed to be the best connector to Flink as
+the storage for streaming warehouse. It uses a full Log-Structured Merge-Tree
+(LSM) structure for high speed and large amount of data update & query capability.
+
+The Table Store supports the following usage:
+- **Streaming Insert**: Write changelog streams, including cdc from database and streams from Flink Sink.
```
Comment (LadyForest) [inline docs/content/_index.md:33]: from Flink Sink? should it either be 
`Write changelog streams, including CDC from database and streams from Flink Source` 
or 
`Write changelog streams, including CDC from database and streams to Flink Sink`

p.s. `cdc`better be capitalized

Comment (JingsongLi) [reply docs/content/_index.md:33]: Let's remove `from Flink Sink`

**Review by LadyForest — APPROVED** [review]
Looks good to me, thanks for the update!

**Pull request merged**